fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -71,7 +67,7 @@ def self.is_supported?(platform)
 
       def self.example_code
         [
-          'git_commit(path: "./version.txt", message: "Version Bump")',
+          #'git_commit(path: "./version.txt", message: "Version Bump")',
```

Comment:
Force to disable this otherwise `test_sample_code` is failing...
I am thinking to revisit this file for proper fix: https://github.com/fastlane/fastlane/blob/master/fastlane/actions/test_sample_code.rb

<img width="766" alt="Screenshot 2021-06-13 at 16 52 12" src="https://user-images.githubusercontent.com/5364500/121812870-e032f380-cc69-11eb-8f12-612d206e4c19.png">